### PR TITLE
haystack_bio

### PR DESCRIPTION
**Pull request by rfarouni**
@bioconda/core

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.

### CODE REVIEW

File: recipes/haystack_bio/build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```
Comment (bgruening) [inline recipes/haystack_bio/build.sh]: Can you please use:
`$PYTHON setup.py install --single-version-externally-managed --record=record.txt`

File: recipes/haystack_bio/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    - ucsc-bigwigaverageoverbed
+    - ucsc-bedgraphtobigwig
+    - ghostscript ==9.10
+    - bx-python 
+    - numpy 
+    - scipy 
+    - matplotlib 
+    - argparse 
+    - jinja2 
+    - nose
```
Comment (bgruening) [inline recipes/haystack_bio/meta.yaml]: is nose really needed during runtime?

File: recipes/haystack_bio/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    - bedtools
+    - sambamba
+    - ucsc-bigwigaverageoverbed
+    - ucsc-bedgraphtobigwig
+    - ghostscript ==9.10
+    - bx-python 
+    - numpy 
+    - scipy 
+    - matplotlib 
+    - argparse
```
Comment (bgruening) [inline recipes/haystack_bio/meta.yaml]: argparse should be in the stdlib

File: recipes/haystack_bio/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    summary: Epigenetic Variability and Transcription Factor Motifs Analysis Pipeline
+
+requirements:
+
+  build:
+    - python
+    - bedtools
+    - openjdk
+    - meme
+    - sambamba
```
Comment (bgruening) [inline recipes/haystack_bio/meta.yaml]: Are sambamba and openjdk ... needed during installation?

Comment (rfarouni) [inline recipes/haystack_bio/meta.yaml]: @bgruening I have updated the recipe. Thanks for catching the redundancies!

**Comment by rfarouni** [comment]
@bgruening I changed the test command to run a bit faster.

**Comment by bgruening** [comment]
What is setup_bioconda doing? You old recipe was fine, I was just trying to get rid of a few more build-dependencies ... sorry for that.

**Comment by rfarouni** [comment]
@bgruening no problem! Thanks for your help. I  needed to have the option for users to install the package manually using pip, so I created another setup.py file. The problem is that I cannot include the required packages in the setup.py file. When I add the following in setup I get an error.
```
        install_requires=[
            'numpy>=1.8.2',
            'pandas>=0.13.1',
            'matplotlib>=1.3.1',
            'scipy>=0.13.3',
            'jinja2>=2.7.3',
            'bx-python>=0.7.3',
             'pybedtools>=0.7.10']

```

**Comment by bgruening** [comment]
Sorry, I think I unnecessary confused you. If you have build dependencies, like numpy and pandas you need to include them. If you only have run-time dependencies our recipe as it is it totally fine!

**Comment by rfarouni** [comment]
@bgruening I see.  But how to find out which packages are build dependencies? If they have Cython code?

**Comment by rfarouni** [comment]
@bgruening I have added back the packages to the build dependencies section so I can use the setup.py for manual install as well.

**Comment by bgruening** [comment]
Thanks a bunch and sorry for the confusion!

**Pull request merged**